src/providers/fault-count/fault-count.ts

```diff
@@ -75,7 +75,7 @@ export class FaultCountProvider {
     return 0;
   }
 
-  private getVehicleChecksFaultCountCatBE = (vehicleChecks: CatBEUniqueTypes.VehicleChecks): VehicleChecksScore => {
+  public getVehicleChecksFaultCountCatBE = (vehicleChecks: CatBEUniqueTypes.VehicleChecks): VehicleChecksScore => {
```

Comment:
Another reference to `CatBE`. Also, does anyone know when and why this was changed from `private` to `public`? Not saying it's wrong, just curious.

Reply:
I'm not sure I'm pretty sure this should still be private I'll change it back

Reply:
Check the references i think it may be used elsewhere (in fact I may have been the one that changed it)
, if you make it private again it won't be accessable.

Reply:
I think it was initially but has been changed so its now called through getVehicleChecksFaultCount so it should be fine but i'll check first